config/routes.yaml

```diff
@@ -1,38 +1,38 @@
 healthCheck:
-  path: '%app.route_prefix%/healthcheck'
+  path: '%app.route_prefix%/v1/healthcheck'
```

Comment:
Suggestion: instead of add v1 into all the routes, we could create another variable app.route_prefix_v1.

Reply:
Yes I can do this and no I won't do this, since this would mean that in case we are changing version of the code under one of the routes - we are changing for all of them, and literally means the same if we would still have version inside `APP_ROUTE_PREFIX` variable.